examples/async-hello-world/async-hello-world.go

```diff
@@ -30,68 +30,46 @@ import (
 func main() {
 
 	// create one connection, save the async results and wait on them explicitly.
-	conn1, err := voltdbclient.OpenConn("localhost:21212")
+	conn, err := voltdbclient.OpenConn([]string{"localhost:21212"})
 	if err != nil {
```

Comment:
This changes OpenConn to accept []string instead of string. 
Should we split the server list string for them? (java client does this)